docs/common/get-started/persistence.mdx

```diff
@@ -0,0 +1,37 @@
+---
+title: "Persistence"
+---
+
+## Persistence
+
+HelloCSV supports using [IndexedDB](https://developer.mozilla.org/en-US/docs/Web/API/IndexedDB_API) for persisting state. This ensures that users don't lose any edits they make if the page refreshes, and allows them to come back to a partially imported state later.
+
+The data is saved in the browsers IndexedDB store, so this data never leaves the user's browser. Therefore, if a different browser is used, the saved session will not be maintained.
+
+HelloCSV will invalidate any state that is saved if the definition of the importer changes. For example, if the definition of an importer is changed such that a new column is added, the state previously saved will not be maintained.
+
+### Persistence Configuration
+
+```tsx
+export interface IndexDBConfig {
+  enabled: boolean;
+  customKey?: string;
+}
+```
+
+
+### Usage
+
+```tsx
+<Importer
+  ...
+  persistenceConfig={{ enabled: true, customKey: `${currentUser.id}-import` }}
```

Comment:
I am not sure this can be common since the field name is different for v.0.1.4, v.0.2.0 and thew new v0.3.0